sway-lib-std/src/intrinsics.sw

```diff
@@ -10,3 +10,39 @@ pub fn is_reference_type<T>() -> bool {
 pub fn size_of<T>() -> u64 {
     __size_of::<T>()
 }
+
+/// Returns the size of a value in bytes.
+pub fn size_of_val<T>(val: T) -> u64 {
+    __size_of_val::<T>(val)
+}
+
+/// Returns the address of the given value.
+pub fn addr_of<T>(val: T) -> u64 {
+    // TODO: Replace with intrinsic: https://github.com/FuelLabs/sway/issues/855
+    if !__is_reference_type::<T>() {
+        // std::revert not available here
```

Comment:
Why can't you import revert?

Reply:
I get an error. My understanding is that the ordering where both `intrinsics` and `revert` are imported matters and `intrinsics` comes before `revert`.

Reply:
You can change the order of `dep`s to put `revert` first, since it doesn't depend on intrinsics.